Main/main.cpp

```diff
@@ -17,6 +17,8 @@
 
 #define TARGET(a) CONCAT(DEVICE_TARGET, a)
 
+const TinyCLR_Api_Provider* globalApiProvider;
```

Comment:
Make a separate PR for adding globalApiProvider to main and change its name to "apiProvider".